common/src/main/java/com/wynntils/core/features/properties/EventListener.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.properties;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to indicate that a feature must be registered to handle events.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface EventListener {}
```

Comment:
Maybe rename to `EventBusSubscriber`? (I know Forge uses this name too, but that does not mean we can't, and it also feels much nicer imo)

Reply:
i'll admit i didn't put much consideration towards debug features. for user features, i was thinking that (eventual) config value could do all the work, but now that you've pointed it out i realize that would require every feature to be enabled by default. i'll make the start enabled stuff universal to features.

Reply:
I think "listener" is a better word. Not only is it much shorter than "bus subscriber", but I also think it hides information at the right level. When you subscribe to these events, you don't really care if they are delivered by a bus or whatever. That's an implementation detail.